db/schema.rb

```diff
@@ -168,6 +168,17 @@
 
   add_index "pages", ["slug"], name: "index_pages_on_slug", unique: true, using: :btree
 
+  create_table "sponsorships", force: :cascade do |t|
+    t.integer  "event_id",        null: false
```

Comment:
The absence of `id` for sponsorship breaks the sponsorship dashboard... Can you add it?